boltdb/adaptor.go

```diff
@@ -0,0 +1,66 @@
+package boltdb
+
+import (
+	"context"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/projectionkit/internal/identity"
+	bolt "go.etcd.io/bbolt"
+)
+
+// adaptor is an implementation adapts an boltdb.ProjectionMessageHandler to the
```

Comment:
```suggestion
// adaptor adapts a boltdb.ProjectionMessageHandler to the
```

This line was copied from `sql.adaptor` which had shoddy docs at the time you branched.